appcues/src/main/java/com/appcues/ui/utils/ActivityExt.kt

```diff
@@ -45,31 +33,15 @@ internal fun Activity.getParentView(): ViewGroup {
             val getRootView: Method = windowManagerClass.getMethod("getRootView", String::class.java)
             val rootViewNames = getViewRootNames.invoke(windowManager) as Array<Any?>
             val rootViews = rootViewNames.map { getRootView(windowManager, it) as View }
-            rootViews.last()
-        } catch (_: Exception) {
+            rootViews.findTopMost() ?: window.decorView
+        } catch (ex: Exception) {
+            Log.e("Appcues", "error getting decorView, ${ex.message}")
```

Comment:
added Log.e similar to ScreenCapture Compose reflection

Reply:
it is awkward to log in these unexpected edge cases with Logcues since it would require passing that reference through complex call chains. I agree that something is better than nothing here, and it is expected to be very rare, so good to add.